examples/plot_stack_masks.py

```diff
@@ -0,0 +1,131 @@
+"""
+Mask and Plot Remote Sensing Data with EarthPy
+==============================================
+
+
+"""
+
+###############################################################################
+# Plotting with EarthPy
+# ---------------------
+#
+# .. note:: Below we walk through a typical workflow using Landsat data with EarthPy.
+
+
+###############################################################################
+# Import Packages
+# ------------------------------
+#
+# To begin, import the needed packages. You will use a combination of several EarthPy
+# modules including spatial, plot and mask.
+
+from glob import glob
+import os
+import matplotlib.pyplot as plt
+import rasterio as rio
+from rasterio.plot import plotting_extent
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+import earthpy.mask as em
+
+# Get data and set your home working directory
+data = et.data.get_data("cold-springs-fire")
```

Comment:
Worth noting here that this command will (somewhat rudely) write data to `$HOME/earth-analytics` and create the directory if it doesn't already exist. 

A better alternative would be to set the path to a temporary directory (e.g., via the tempfile library), and include a commented out line of code that shows users how they would set the path to a custom directory. 

CRAN's take on this in their guidelines prioritizes respect for the user's filesystem:

"Packages should not write in the user’s home filespace (including clipboards), nor anywhere else on the file system apart from the R session’s temporary directory (or during installation in the location pointed to by TMPDIR:  and such usage shouldbe cleaned up).  Installing into the system’s R installation (e.g., scripts to its bin directory) is not allowed. Limited exceptions may be allowed in interactive sessions if the package obtains confirmation from the user."

Reply:
In this case, we've set this up to ensure students have success in our class. we used the home directory as a space that all users have and also that can be used on the hub. i wouldn't want to change this behavior. Also it should download to `HOME/earth-analytics/data` if it's working properly. 

This is supposed to showcase the default behavior. if the users don't do this, none of our lesson code will work by default given varying paths. 

i could see us having a download data vignette that provides more info about downloading data to varying paths but that wouldn't fit in a plot vignette.it would be very distracting